### ISSUE TITLE
Discussion: Related, forked, and hybrid cultures

### ISSUE THREAD

**Issue by MilesBHuff** [TownyAdvanced/TownyCultures#21]
In real life, similar cultures often share affinities for each other.  For example, the Swedes, Norwegians, and Danes share common heritage, values, and languages.  These cultures all split off from a single parent culture, Norse.
Similarly, English is a unique mixture of the Saxon, Celtic, Norman, and Norse cultures, and shares affinities to each.  American culture is an offshoot of English culture, with different emphases.

I'm not sure how these realities could be represented in-game;  but one thing that might be interesting is allowing people to fork a culture -- to create a derivative of a widespread culture -- or to create a hybrid culture -- to blend two cultures into one.

There could be a command to show a textual tree of cultures in chat.  Since one culture could have more than one parent, the command would require an argument:  namely, the culture whose parents and children to display.

As for impacts on gameplay, there are several possibilities.
One is that, if #20 is implemented, child cultures could split their parents' bonuses -- so if parent A has the desert bonus and parent B has the ocean bonus, then the child culture would have half the desert bonus and half the ocean bonus.  The overall strength of the bonus would still scale by the criteria in #18.

**Comment by LlmDl** [comment]
Sounds like an awful lot of work, even more than the origins ticket. There's nothing stopping a town from deciding its a hydrid of two cultures and just changing their culture to signify this.

**Comment by MilesBHuff** [comment]
Yeah, it's a huge feature.  The kind of thing I figured I might as well make an issue ticket for, but the kind of thing I have almost of hopes of seeing implemented.
